### PR TITLE
added warning and changed removal type to advance for …

### PR DESCRIPTION
**Pull request by gs1719**
…oshare

Since it break the functionality of sharing the screenshot directly may disturb some user

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -20133,14 +20133,23 @@
     "removal": "Recommended"
   },
   {
-    "id": "com.coloros.oshare",
+    "id": "com.heytap.market",
     "list": "Oem",
-    "description": "Oppo Share\nFile sharing app to transfer data from/to Oppo devices only. Seems to use weak crypto (AES ECB mode) and has weird permissions (READ_CONTACTS ? Really ?).\n\nPithus analysis: https://beta.pithus.org/report/170f4a14be24a2e2135cd956a038aae9e2f78c845f3161b84c5545dbec03fad9",
+    "description": "Oppo application Market .Safe to remove unless you are using it",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json]:
```suggestion
    "description": "Oppo application Market. Safe to remove unless you are using it.\nhttps://developers.oppomobile.com/newservice/capability?pagename=app_store",
```

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -20133,14 +20133,23 @@
...
     "dependencies": [],
     "neededBy": [],
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.coloros.oshare",
+    "list": "Oem",
+    "description": "Oppo Share\nFile sharing app to transfer data from/to Oppo devices only. Seems to use weak crypto (AES ECB mode) and has weird permissions (READ_CONTACTS ? Really ?).\n[Warning]May break the functionality to share photos from oem photos app only and share with send prompt after screenshot is not working(though screenshot is saved and could be send through different similar application) \n\nPithus analysis: https://beta.pithus.org/report/170f4a14be24a2e2135cd956a038aae9e2f78c845f3161b84c5545dbec03fad9",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json]:
```suggestion
    "description": "Oppo Share\nFile sharing app to transfer data from/to Oppo devices only. Seems to use weak crypto (AES ECB mode) and has weird permissions (such as `READ_CONTACTS`).\n[Warning]May break the functionality to share photos from oem photos app only and share with send prompt after screenshot is not working (though screenshot is saved and could be send through different similar application)\nPithus analysis: https://beta.pithus.org/report/170f4a14be24a2e2135cd956a038aae9e2f78c845f3161b84c5545dbec03fad9",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Comment by AnonymousWP** [comment]
@gs1719 Why are you closing it?? It needs to get merged by @0x192 first.

**Comment by gs1719** [comment]
Actually I am new in this 
Not able to understand what to do with suggestions so that's why I close it 
Should I reopen it or what to do to make changes?

**Comment by AnonymousWP** [comment]
This is how you do it, and you're not supposed to close my suggestions: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request. Don't close the pull request either, but commit the suggestions I've made.

**Review by AnonymousWP — APPROVED** [review]

**Comment by 0x192** [comment]
Sorry for the lag! It's finally merged! :slightly_smiling_face:

**Pull request merged**